v3/src/components/graph/adornments/v2-adornment-importer.ts

```diff
@@ -128,21 +153,31 @@ const instanceKeysForAdornments = (props: IInstanceKeysForAdornmentsProps) => {
   return instanceKeys
 }
 
+export type AdornmentAttributeDescriptions = Partial<Record<GraphAttrRole, IAttributeDescriptionSnapshot>>
```

Comment:
`AdornmentAttributeDescriptions` aren't adornment specific, so I'd be inclined to call it something like `GraphAttributeDescriptions` and move the definition into `data-display-types.ts` or `graphing-types.ts`.